website/cue/reference/configuration.cue

```diff
@@ -145,6 +145,62 @@ configuration: {
 				syntax: "literal"
 			}
 		}
+
+		proxy: {
+			common:      false
+			description: "Configures an HTTP(S) proxy for Vector to use."
+			required:    false
+			type: object: options: {
+				enabled: {
+					common:      false
+					description: "If false the proxy will be disabled."
+					required:    false
+					type: bool: default: true
+				}
+				http: {
+					common:      false
+					description: "The URL to proxy HTTP requests through."
+					required:    false
+					type: string: {
+						default: null
+						examples: ["http://foo.bar:3128"]
+						syntax: "literal"
+					}
+				}
+				https: {
+					common:      false
+					description: "The URL to proxy HTTPS requests through."
+					required:    false
+					type: string: {
+						default: null
+						examples: ["http://foo.bar:3128"]
```

Comment:
```suggestion
						examples: ["https://foo.bar:3128"]
```
?

Reply:
This was actually discussed in the original PR too 😄 It can be common to use `http://` even for proxying HTTPS traffic to avoid having to setup a certificate for the proxy.